tui/kernel_cmdline.go

```diff
@@ -5,36 +5,66 @@
 package tui
 
 import (
+	"fmt"
+	"strings"
+
 	"github.com/VladimirMarkelov/clui"
 )
 
 // KernelCMDLine is the Page implementation for the kernel cmd line configuration page
 type KernelCMDLine struct {
 	BasePage
-	kernelCMDLineEdit *clui.EditField
+	addKernelArgEdit *clui.EditField
+	remKernelArgEdit *clui.EditField
 }
 
 // GetConfiguredValue Returns the string representation of currently value set
 func (pp *KernelCMDLine) GetConfiguredValue() string {
-	cmdLine := pp.getModel().KernelCMDLine
+	result := ""
+
+	if pp.getModel().KernelArguments != nil {
+		values := []string{}
+
+		addKernelArgs := strings.Join(pp.getModel().KernelArguments.Add, " ")
+		remKernelArgs := strings.Join(pp.getModel().KernelArguments.Remove, " ")
+
+		if addKernelArgs != "" {
+			values = append(values, fmt.Sprintf("add: %s", addKernelArgs))
```

Comment:
Maybe make the labels "ADD:" and "REMOVE:"? Or at least "Add:" and "Remove:"

Reply:
done